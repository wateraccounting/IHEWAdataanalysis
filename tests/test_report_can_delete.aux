\relax 
\providecommand\hyper@newdestlabel[2]{}
\providecommand\HyperFirstAtBeginDocument{\AtBeginDocument}
\HyperFirstAtBeginDocument{\ifx\hyper@anchor\@undefined
\global\let\oldcontentsline\contentsline
\gdef\contentsline#1#2#3#4{\oldcontentsline{#1}{#2}{#3}}
\global\let\oldnewlabel\newlabel
\gdef\newlabel#1#2{\newlabelxx{#1}#2}
\gdef\newlabelxx#1#2#3#4#5#6{\oldnewlabel{#1}{{#2}{#3}}}
\AtEndDocument{\ifx\hyper@anchor\@undefined
\let\contentsline\oldcontentsline
\let\newlabel\oldnewlabel
\fi}
\fi}
\global\let\hyper@last\relax 
\gdef\HyperFirstAtBeginDocument#1{#1}
\providecommand\HyField@AuxAddToFields[1]{}
\providecommand\HyField@AuxAddToCoFields[2]{}
\abx@aux@refcontext{ynt/global//global/global}
\@writefile{toc}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lof}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{lot}{\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax }
\@writefile{toc}{\contentsline {section}{List of Figures}{i}{section*.2}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{List of Tables}{ii}{section*.3}\protected@file@percent }
\newlabel{sec:Acknowledgements}{{}{iii}{Acknowledgements}{section*.4}{}}
\@writefile{toc}{\contentsline {section}{Acknowledgements}{iii}{section*.4}\protected@file@percent }
\gdef \LT@i {\LT@entry 
    {4}{209.12685pt}\LT@entry 
    {4}{209.12685pt}}
\newlabel{sec:AbbreviationsandAcronyms}{{}{iv}{Abbreviations and Acronyms}{section*.5}{}}
\@writefile{toc}{\contentsline {section}{Abbreviations and Acronyms}{iv}{table.1}\protected@file@percent }
\newlabel{sec:Executivesummary}{{}{v}{Executive summary}{section*.6}{}}
\@writefile{toc}{\contentsline {section}{Executive summary}{v}{section*.6}\protected@file@percent }
\@writefile{toc}{\contentsline {section}{\numberline {1}Introduction}{1}{section.1}\protected@file@percent }
\newlabel{sec:Introduction}{{1}{1}{Introduction}{section.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {1.1}Subsection 1.1}{1}{subsection.1.1}\protected@file@percent }
\newlabel{subsec:Subsection1.1}{{1.1}{1}{Subsection 1.1}{subsection.1.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {2}Methodology}{2}{section.2}\protected@file@percent }
\newlabel{sec:Methodology}{{2}{2}{Methodology}{section.2}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {2.1}WaPOR database}{2}{subsection.2.1}\protected@file@percent }
\newlabel{subsec:WaPORdatabase}{{2.1}{2}{WaPOR database}{subsection.2.1}{}}
\@writefile{toc}{\contentsline {section}{\numberline {3}Test from YAML}{3}{section.3}\protected@file@percent }
\newlabel{sec:TestfromYAML}{{3}{3}{Test from YAML}{section.3}{}}
\abx@aux@cite{bertram}
\abx@aux@segm{0}{0}{bertram}
\abx@aux@cite{simon06}
\abx@aux@segm{0}{0}{simon06}
\@writefile{toc}{\contentsline {section}{\numberline {4}Test}{4}{section.4}\protected@file@percent }
\newlabel{sec:Test}{{4}{4}{Test}{section.4}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.1}Reference}{4}{subsection.4.1}\protected@file@percent }
\newlabel{subsec:Reference}{{4.1}{4}{Reference}{subsection.4.1}{}}
\abx@aux@page{1}{4}
\abx@aux@page{2}{4}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.2}List}{4}{subsection.4.2}\protected@file@percent }
\newlabel{subsec:List}{{4.2}{4}{List}{subsection.4.2}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.3}Figure}{4}{subsection.4.3}\protected@file@percent }
\newlabel{subsec:Figure}{{4.3}{4}{Figure}{subsection.4.3}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.4}Talbe}{4}{subsection.4.4}\protected@file@percent }
\newlabel{subsec:Talbe}{{4.4}{4}{Talbe}{subsection.4.4}{}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{Caption Table}}{4}{table.2}\protected@file@percent }
\newlabel{table:tab1}{{2}{4}{Caption Table}{table.2}{}}
\gdef \LT@ii {\LT@entry 
    {1}{50.05pt}\LT@entry 
    {1}{49.65001pt}\LT@entry 
    {1}{49.65001pt}}
\@writefile{lot}{\contentsline {table}{\numberline {2}{Caption Table}}{5}{table.2}\protected@file@percent }
\newlabel{table:tab1}{{2}{5}{Caption Table}{table.2}{}}
\@writefile{lof}{\contentsline {figure}{\numberline {1}{\ignorespaces Caption Figure}}{6}{figure.1}\protected@file@percent }
\newlabel{figure:fig1}{{1}{6}{Caption Figure}{figure.1}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.5}Equation numpy}{6}{subsection.4.5}\protected@file@percent }
\newlabel{subsec:Equationnumpy}{{4.5}{6}{Equation numpy}{subsection.4.5}{}}
\newlabel{equation:equ1}{{2}{6}{Equation numpy}{equation.4.2}{}}
\@writefile{toc}{\contentsline {subsection}{\numberline {4.6}Equation quantities}{6}{subsection.4.6}\protected@file@percent }
\newlabel{subsec:Equationquantities}{{4.6}{6}{Equation quantities}{subsection.4.6}{}}
\newlabel{equation:equ2}{{3}{6}{Equation quantities}{equation.4.3}{}}
\newlabel{sec:References}{{4.6}{7}{References}{section*.7}{}}
\abx@aux@page{3}{7}
\abx@aux@page{4}{7}
\@writefile{toc}{\contentsline {section}{References}{7}{section*.7}\protected@file@percent }
\newlabel{sec:Annexes}{{4.6}{8}{Annexes}{section*.8}{}}
\@writefile{toc}{\contentsline {section}{Annexes}{8}{section*.8}\protected@file@percent }
\newlabel{LastPage}{{}{8}{}{page.8}{}}
\xdef\lastpage@lastpage{8}
\xdef\lastpage@lastpageHy{8}
\abx@aux@refcontextdefaultsdone
\abx@aux@defaultrefcontext{0}{bertram}{ynt/global//global/global}
\abx@aux@defaultrefcontext{0}{simon06}{ynt/global//global/global}
