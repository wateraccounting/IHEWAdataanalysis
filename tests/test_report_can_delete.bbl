% $ biblatex auxiliary file $
% $ biblatex bbl format version 3.1 $
% Do not modify the above lines!
%
% This is an auxiliary file used by the 'biblatex' package.
% This file may safely be deleted. It will be recreated by
% biber as required.
%
\begingroup
\makeatletter
\@ifundefined{ver@biblatex.sty}
  {\@latex@error
     {Missing 'biblatex' package}
     {The bibliography requires the 'biblatex' package.}
      \aftergroup\endinput}
  {}
\endgroup


\refsection{0}
  \datalist[entry]{ynt/global//global/global}
    \entry{bertram}{article}{}
      \name{author}{2}{}{%
        {{un=0,uniquepart=base,hash=8a5b475249c01d8aa99f2275128c1037}{%
           family={Bertram},
           familyi={B\bibinitperiod},
           given={Aaron},
           giveni={A\bibinitperiod},
           givenun=0}}%
        {{un=0,uniquepart=base,hash=60f0568f0258e96055aef41b438ecbd2}{%
           family={Wentworth},
           familyi={W\bibinitperiod},
           given={Richard},
           giveni={R\bibinitperiod},
           givenun=0}}%
      }
      \strng{namehash}{4be7c21774d86b9e1f65ca280181467e}
      \strng{fullhash}{4be7c21774d86b9e1f65ca280181467e}
      \strng{bibnamehash}{4be7c21774d86b9e1f65ca280181467e}
      \strng{authorbibnamehash}{4be7c21774d86b9e1f65ca280181467e}
      \strng{authornamehash}{4be7c21774d86b9e1f65ca280181467e}
      \strng{authorfullhash}{4be7c21774d86b9e1f65ca280181467e}
      \field{sortinit}{1}
      \field{sortinithash}{50c6687d7fc80f50136d75228e3c59ba}
      \field{extradatescope}{labelyear}
      \field{labeldatesource}{}
      \field{labelnamesource}{author}
      \field{labeltitlesource}{shorttitle}
      \field{annotation}{An \texttt{article} entry with a \texttt{volume} and a \texttt{number} field}
      \field{journaltitle}{J.~Amer. Math. Soc.}
      \field{langid}{english}
      \field{langidopts}{variant=american}
      \field{number}{2}
      \field{shorttitle}{Gromov invariants}
      \field{title}{Gromov invariants for holomorphic maps on {Riemann} surfaces}
      \field{volume}{9}
      \field{year}{1996}
      \field{dateera}{ce}
      \field{pages}{529\bibrangedash 571}
      \range{pages}{43}
    \endentry
    \entry{simon06}{article}{}
      \name{author}{5}{}{%
        {{un=0,uniquepart=base,hash=12aaa11594aa9fb02370795d6385ff1e}{%
           family={Simon},
           familyi={S\bibinitperiod},
           given={E.},
           giveni={E\bibinitperiod},
           givenun=0}}%
        {{un=0,uniquepart=base,hash=25864f66a618652e1eb89ef82f3486d3}{%
           family={Flaschker},
           familyi={F\bibinitperiod},
           given={N.},
           giveni={N\bibinitperiod},
           givenun=0}}%
        {{un=0,uniquepart=base,hash=49938d63f29583e5707df0547c66b421}{%
           family={Schadewaldt},
           familyi={S\bibinitperiod},
           given={P.},
           giveni={P\bibinitperiod},
           givenun=0}}%
        {{un=0,uniquepart=base,hash=9e3cd84c0bb3a106318d9b907de3cbdf}{%
           family={Langenbeck},
           familyi={L\bibinitperiod},
           given={U.},
           giveni={U\bibinitperiod},
           givenun=0}}%
        {{un=0,uniquepart=base,hash=b3fc4f414db6edfb9ef06613bbf1e07a}{%
           family={Wendel},
           familyi={W\bibinitperiod},
           given={U.},
           giveni={U\bibinitperiod},
           givenun=0}}%
      }
      \strng{namehash}{718e5cc4751acb96811b6c7363edc39a}
      \strng{fullhash}{f3b35e818a98e417c83b477140741ab6}
      \strng{bibnamehash}{718e5cc4751acb96811b6c7363edc39a}
      \strng{authorbibnamehash}{718e5cc4751acb96811b6c7363edc39a}
      \strng{authornamehash}{718e5cc4751acb96811b6c7363edc39a}
      \strng{authorfullhash}{f3b35e818a98e417c83b477140741ab6}
      \field{sortinit}{2}
      \field{sortinithash}{ed39bb39cf854d5250e95b1c1f94f4ed}
      \field{extradatescope}{labelyear}
      \field{labeldatesource}{year}
      \field{labelnamesource}{author}
      \field{labeltitlesource}{title}
      \field{journaltitle}{Journal of inherited metabolic disease}
      \field{number}{6}
      \field{title}{Variant maple syrup urine disease (MSUD)--the entire spectrum}
      \field{volume}{29}
      \field{year}{2006}
      \field{pages}{716\bibrangedash 24}
      \range{pages}{9}
      \verb{doi}
      \verb 10.1007/s10545-006-0276-1
      \endverb
    \endentry
  \enddatalist
\endrefsection
\endinput

