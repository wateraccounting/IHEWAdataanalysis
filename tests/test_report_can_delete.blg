[0] Config.pm:304> INFO - This is Biber 2.14
[0] Config.pm:307> INFO - Logfile is 'test_report_can_delete.blg'
[77] biber-MSWIN64:322> INFO - === 
[107] Biber.pm:415> INFO - Reading 'test_report_can_delete.bcf'
[209] Biber.pm:943> INFO - Found 2 citekeys in bib section 0
[228] Biber.pm:4256> INFO - Processing section 0
[228] Utils.pm:75> INFO - Globbing data source 'test_report.bib'
[244] Utils.pm:91> INFO - Globbed data source 'test_report.bib' to test_report.bib
[264] Biber.pm:4455> INFO - Looking for bibtex format file 'test_report.bib' for section 0
[276] bibtex.pm:1653> INFO - LaTeX decoding ...
[669] bibtex.pm:1471> INFO - Found BibTeX data source 'test_report.bib'
[744] UCollate.pm:68> INFO - Overriding locale 'en-US' defaults 'normalization = NFD' with 'normalization = prenormalized'
[744] UCollate.pm:68> INFO - Overriding locale 'en-US' defaults 'variable = shifted' with 'variable = non-ignorable'
[744] Biber.pm:4084> INFO - Sorting list 'ynt/global//global/global' of type 'entry' with template 'ynt' and locale 'en-US'
[744] Biber.pm:4090> INFO - No sort tailoring available for locale 'en-US'
[749] bbl.pm:652> INFO - Writing 'test_report_can_delete.bbl' with encoding 'UTF-8'
[751] bbl.pm:755> INFO - Output to test_report_can_delete.bbl
