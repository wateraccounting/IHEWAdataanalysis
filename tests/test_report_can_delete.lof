\boolfalse {citerequest}\boolfalse {citetracker}\boolfalse {pagetracker}\boolfalse {backtracker}\relax 
\contentsline {figure}{\numberline {1}{\ignorespaces Caption Figure}}{6}{figure.1}%
