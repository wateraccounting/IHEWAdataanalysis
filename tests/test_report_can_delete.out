\BOOKMARK [1][-]{section*.2}{List of Figures}{}% 1
\BOOKMARK [1][-]{section*.3}{List of Tables}{}% 2
\BOOKMARK [1][-]{section*.4}{Acknowledgements}{}% 3
\BOOKMARK [1][-]{table.1}{Abbreviations and Acronyms}{}% 4
\BOOKMARK [1][-]{section*.6}{Executive summary}{}% 5
\BOOKMARK [1][-]{section.1}{Introduction}{}% 6
\BOOKMARK [2][-]{subsection.1.1}{Subsection 1.1}{section.1}% 7
\BOOKMARK [1][-]{section.2}{Methodology}{}% 8
\BOOKMARK [2][-]{subsection.2.1}{WaPOR database}{section.2}% 9
\BOOKMARK [1][-]{section.3}{Test from YAML}{}% 10
\BOOKMARK [1][-]{section.4}{Test}{}% 11
\BOOKMARK [2][-]{subsection.4.1}{Reference}{section.4}% 12
\BOOKMARK [2][-]{subsection.4.2}{List}{section.4}% 13
\BOOKMARK [2][-]{subsection.4.3}{Figure}{section.4}% 14
\BOOKMARK [2][-]{subsection.4.4}{Talbe}{section.4}% 15
\BOOKMARK [2][-]{subsection.4.5}{Equation numpy}{section.4}% 16
\BOOKMARK [2][-]{subsection.4.6}{Equation quantities}{section.4}% 17
\BOOKMARK [1][-]{section*.7}{References}{}% 18
\BOOKMARK [1][-]{section*.8}{Annexes}{}% 19
